src/components/join-us-tab/join-us-tab.tsx

```diff
@@ -32,8 +32,9 @@ export default function JoinUsTab() {
               <TabList
                 orientation={smallScreen ? 'vertical' : 'horizontal'}
                 onChange={handleChange}
-                centered
+                centered={true}
                 indicatorColor="primary"
+                variant={smallScreen ? 'fullWidth' : 'standard'}
```

Comment:
I don't think it's this change specifically, but it seemed like the most relevant part of the diff to bring it up

It seems like the `TabList` component renders okay on `iPhone 14 Pro Max` at a width of `430px`:

![image](https://github.com/CodeForBc/codeforbc-web/assets/16601729/df2097cd-45ef-4780-ad4b-66489ad9eb17)

But the text starts to get cut off at `iPhone 12 Pro` (`390px`) and smaller

![image](https://github.com/CodeForBc/codeforbc-web/assets/16601729/2739b24d-0cd9-49b4-a861-f099cfd96a1a)

`iPhone SE` (`375px`)

![image](https://github.com/CodeForBc/codeforbc-web/assets/16601729/fd32e7b3-5329-476f-859f-36f534ad151d)

Reply:
addressed in [3f13822](https://github.com/CodeForBc/codeforbc-web/pull/16/commits/3f1382206e1bec2f3698346a94c08cf2acc4b4b1)